src/core/chatRoster.js

```diff
@@ -31,6 +38,20 @@ Candy.Core.ChatRoster = function () {
  */
 Candy.Core.ChatRoster.prototype.add = function(user) {
 	this.items[user.getJid()] = user;
+
+  var evtData = {
+    user: user,
+    roomJid: this.roomJid
+  };
+
+  /** Event: candy:roster.user.after-add
+   * After initialising a room
+   *
+   * Parameters:
+   *   (ChatUser Object) user - the user that was added
+   *   (String) roomJid - the jid of the room the user is being added to
+   */
+  $(Candy).triggerHandler('candy:roster.user.after-add', evtData);
```

Comment:
I wasn't sure if it should be view or not since the roster isn't technically a part of a view--it's a part of core. I considered also `candy:view.room.roster.user.after-add` but thought that that was a bit long and finally settled on just removing the `view.room` bit to avoid any confusion.